<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>229</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_network">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_network_card">
       <attribute name="title">
        <string>Network Card</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="cbx_interface_0">
          <property name="text">
           <string>Intel(R) Wireless-AC 9560 160MHz</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbx_interface_1">
          <property name="text">
           <string>TP-Link Wireless USB Adapter</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="cbx_interface_2">
          <property name="text">
           <string>Realtek RTL8192CU Wireless LAN 802.11n USB 2.0 Network Adapter</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="cbx_interface_3">
          <property name="text">
           <string>Realtek RTL8821CE 802.11ac PCIe Adapter</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btn_get_status">
          <property name="text">
           <string>Get Status</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_wifi_setting">
       <attribute name="title">
        <string>Wifi Setting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>SSID :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Password :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="txb_ssid"/>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="txb_password"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btn_save_wifi_setting">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_wifi_on">
          <property name="text">
           <string>Wifi On</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="btn_wifi_off">
          <property name="text">
           <string>Wifi Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
